docker-wrappers/BowTieBuilder/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.8-bullseye
+
+WORKDIR /btb
+RUN wget https://raw.githubusercontent.com/Reed-CompBio/BowTieBuilder-Algorithm/main/btb.py
```

Comment:
I wonder if we should have this be a specific commit hash, in case we update `btb.py`? Maybe not something for us to worry about now - at least this way if we ever to update `btb.py` this Dockerfile will build a new image with the most recent version of `btb.py`.

Reply:
I'll leave as-is, and we can go back if we change our minds? I'm also unsure how accessing a specific commit hash would work